### PR TITLE
(DOCS-8553) App Builder - Workflow returning data to app

### PR DESCRIPTION
**Pull request by neko-dd**
### What does this PR do? What is the motivation?

* Moves Queries to its own page
* Adds sample app about Workflow -> App integration

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/service_management/app_builder/queries.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+Similar to UI component events, you can configure a reaction to trigger after a query executes. A **post-query hook** can set a UI component state, open or close a modal, trigger another query, or even run custom JavaScript. For example, the [ECS Task Manager][7] blueprint's `scaleService` query uses a post-query hook to rerun the `describeService` query after it executes.
+
+## Error notifications
+
+To display a toast (a brief notification message) to the user when the system returns an error, toggle **Show Toast on Errors** in the **Advanced** section of a query.
+
+## Confirmation prompts
+
+To prompt a user for confirmation before the query runs, toggle the **Requires Confirmation** option in the **Advanced** section of a query.
```
Comment (neko-dd) [inline content/en/service_management/app_builder/queries.md:66]: Just a note that none of this content is new, it's the content that moved over from build.md

File: content/en/service_management/app_builder/queries.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+## Example app
+
+### Return workflow results to an app
+App Builder queries can trigger Workflow Automation workflows. Apps can then use the results of those workflows.
+
+This app triggers a workflow, which sends a poll to a Slack channel asking the user to pick from one of two options. Based on the option the user chooses, the workflow issues one of two different HTTP GET requests, which then returns data that is displayed in the app.
```
Comment (urseberry) [inline content/en/service_management/app_builder/queries.md:73]: The description mentions two options, but the video appears to only show one button.

File: content/en/service_management/app_builder/queries.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+{{< img src="service_management/app_builder/workflow-trigger-from-app.mp4" alt="Clicking Trigger Workflow polls Slack then returns a random cat or dog fact" video="true" width="486px">}}
+
+{{% collapse-content title="Build the app" level="h4" %}}
+
+##### Create workflow
+
+1. In a new workflow canvas, under **Datadog Triggers**, click **App**. 
+1. Under the **App** trigger step, click the plus (**+**) icon then search for "Make a decision" and select the **Make a decision** Slack action.
```
Comment (urseberry) [inline content/en/service_management/app_builder/queries.md:82]:
```suggestion
1. Under the **App** trigger step, click the plus (**+**) icon, then search for "Make a decision" and select the **Make a decision** Slack action.
```

File: content/en/service_management/app_builder/queries.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+1. Select your workspace and choose a channel to poll.
+1. Fill in the prompt text "Cat fact or dog fact?" and change the button choices to "Cat fact" and "Dog fact".
```
Comment (urseberry) [inline content/en/service_management/app_builder/queries.md:84]:
```suggestion
1. Fill in the prompt text with "Cat fact or dog fact?" and change the button choices to "Cat fact" and "Dog fact".
```

File: content/en/service_management/app_builder/queries.md (hunk 2 above)
Comment (neko-dd) [reply content/en/service_management/app_builder/queries.md:73]: The workflow goes out to Slack, which polls with two options. I don't think it's feasible to include that in the video.

File: content/en/service_management/app_builder/queries.md (hunk 4 above)
Comment (neko-dd) [reply content/en/service_management/app_builder/queries.md:84]: I actually think the original is more correct.

File: content/en/service_management/app_builder/queries.md (hunk 2 above)
Comment (urseberry) [reply content/en/service_management/app_builder/queries.md]: I understand, thanks!

File: content/en/service_management/app_builder/queries.md (hunk 4 above)
Comment (urseberry) [reply content/en/service_management/app_builder/queries.md:84]: I don't feel strongly about my edit and can understand your original, too. 😄

**Review by urseberry — APPROVED** [review]

**Pull request merged**